docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3'
 
 services:
   bot:
-    image: blurplefy
+    build: .
```

Comment:
Changing these from using the tag to `build` leads to docker-compose building the exact same image twice, doubling rebuild time.

Reply:
Strange, when I was initially building it using docker-compose it would re-use the cache from the first image it built (in this case, the worker).

![image](https://user-images.githubusercontent.com/24538037/57741417-941fba00-7671-11e9-80a4-efb5ae3d43d5.png)